lab-ben/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "lab-ben",
+  "version": "1.0.0",
+  "description": "script adding, call and apply testing",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "mocha",
+    "lint": "eslint ./lib",
```

Comment:
Good job remembering your ``lint`` script. No points lost for this because I don't think Scott taught you, but ``eslint`` is actually d dev dependency you'll have to install in order for this script to work. 

If you ``npm i -D eslint`` in your terminal, you'll have the dependency.